server/backend/linux.py

```
@@ -1 +1,600 @@
+        )
+
+    # Check if we need to load a different model
+    if _current_model_path != model_path_str:
+        # Proactively clean up any previously loaded runner to release memory
+        if _model_cache:
+            try:
+                for _old_runner in list(_model_cache.values()):
+                    try:
+                        _old_runner.cleanup()
+                    except Exception:
+                        pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)